config/routes.rb

```diff
@@ -1,8 +1,9 @@
 Rails.application.routes.draw do
-  root "home#index"
-  resources :home, only: [:index, :edit, :update]
-  resources :learn_more, only: [:index, :edit, :update]
-  resources :tools_resources, only: [:index, :edit, :update]
+  root "static_pages#home"
+  get "/home", to: "static_pages#home", as: "home"
+  get "/learn_more", to: "static_pages#learn_more", as: "learn_more"
+  get "/tools_resources", to: "static_pages#tools_resources", as: "tools_resources"
+  get "/locations", to: "static_pages#locations", as: "locations"
   resources :companies, only: [:create]
   get 'signup', to: "companies#new"
 end
```

Comment:
Switched from having each page having it's own controller to one static pages controller. Also removed the edit and update since we are now updating the content in a Content Controller.